requirements.txt

```diff
@@ -29,3 +29,4 @@ wheel~=0.37.1
 setuptools~=60.2.0
 graphviz~=0.20.1
 scipy==1.9.3
+requests~=2.27.1
```

Comment:
Do we need it?

Reply:
nope. i was toying with using some external APIs to get some realistic looking columns, but i scrapped the idea for now